otx/algorithms/classification/adapters/mmcls/models/classifiers/byol.py

```diff
@@ -50,7 +52,11 @@ def __init__(
 
         # build backbone
         self.online_backbone = build_backbone(backbone)
-        self.target_backbone = build_backbone(backbone)
+
+        target_backbone_cfg = copy.deepcopy(backbone)
+        target_backbone_cfg["pretrained"] = None
```

Comment:
Could you explain why the target backbone's pretrain should be False?

Reply:
I detectect mmcv bug that ckpt corruption when consecutive ckpt loading happens. And actually target backbone does not need to load ckpt pretrained weight. So made it not incur the corresponding error.